core/nwb.behavior.yaml

```diff
@@ -38,6 +38,10 @@ groups:
         is 'meters'. Actual stored values are not necessarily stored in these units.
         To access the data in these units, multiply 'data' by 'conversion' and add 'offset'.
       required: false
+    - name: bounds
+      dtype: float32
+      doc: The boundary range for the data. The units are the same as the units for the data.
+      required: false
```

Comment:
What about multi-dimensional SpatialSeries? I think we could add shape constraints, e.g. (None, 2).

Reply:
I agree - we should add shape constraints. (None, 2) is OK but we can be more specific - (1, 2), (2, 2), or (3, 2). The first dimension should match the length of the second dimension of `data`.

Reply:
See suggestion below.